components/ProjectForm/index.js

```diff
@@ -24,8 +20,16 @@ export default function ProjectForm({ onAddProject }) {
     reset,
   } = useForm();
 
-  const onSubmit = (data) => {
-    onAddProject(data);
+  const onSubmit = async (data) => {
+    const request = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(data),
+    };
+
+    const response = await fetch("/api/projects", request);
+    const jsonData = await response.json();
+    mutate();
```

Comment:
you don't need to mutate if you redirect your user to the main page, since this will trigger a refetch of the data automatically.

Reply:
we tried to remove it, but then we always need to click refresh again to see the newly created project.
with mutate() we can see it directly, therefore left it as is.

Reply:
We tried this as well, but then we are not re-directed to the homepage and that would be our preference.